Strict mode doesn't cause failure when images are missing, though it reports that as an ERROR

## Expected Behavior
When processing-mode=strict, builds fail when any missing file is reported (error [DOTX008E][ERROR])

## Actual Behavior
When using strict, I get build failures when a file referenced by a topicref in a ditamap isn't found. But I don't get failures for missing images, although strict mode is supposed to fail on any error. 



## Copy of the error message, log file or stack trace
[dita-ot] [topic-reader] [DOTX008E][ERROR] File 'file:/C:/workgit/dt-pipeline/dita-test/content/images/NONEXISTENTnote.png' does not exist or cannot be loaded.

## Environment
<!-- Include relevant details about the environment you experienced this in. -->

* DITA-OT version: 3.6.1
* Operating system and version: Windows 10, LInux

* How did you run DITA-OT?  command line
* Transformation type: HTML and PDF

--- a/src/main/java/org/dita/dost/module/reader/TopicReaderModule.java
+++ b/src/main/java/org/dita/dost/module/reader/TopicReaderModule.java
@@ -12,12 +12,14 @@ import net.sf.saxon.s9api.QName;
 import net.sf.saxon.s9api.XdmNode;
 import org.dita.dost.exception.DITAOTException;
 import org.dita.dost.exception.DITAOTXMLErrorHandler;
+import org.dita.dost.exception.UncheckedDITAOTException;
 import org.dita.dost.log.MessageUtils;
 import org.dita.dost.module.filter.SubjectScheme;
 import org.dita.dost.pipeline.AbstractPipelineInput;
 import org.dita.dost.pipeline.AbstractPipelineOutput;
 import org.dita.dost.reader.GenListModuleReader.Reference;
 import org.dita.dost.reader.SubjectSchemeReader;
+import org.dita.dost.util.Configuration;
 import org.dita.dost.util.Job.FileInfo;
 import org.dita.dost.writer.DebugFilter;
 import org.dita.dost.writer.NormalizeFilter;
@@ -296,7 +298,11 @@ public final class TopicReaderModule extends AbstractReaderModule {
             } else if (ATTR_FORMAT_VALUE_IMAGE.equals(file.format)) {
                 formatSet.add(file);
                 if (!exists(file.filename)) {
-                    logger.warn(MessageUtils.getMessage("DOTX008E", file.filename.toString()).toString());
+                    if (processingMode == Configuration.Mode.STRICT) {
+                        throw new UncheckedDITAOTException(MessageUtils.getMessage("DOTX008E", file.filename.toString()).toException());
+                    } else {
+                        logger.warn(MessageUtils.getMessage("DOTX008E", file.filename.toString()).toString());
+                    }
                 }
             } else {
                 htmlSet.put(file.format, file.filename);

